app/dashboard/notifications.py

```diff
@@ -69,6 +78,8 @@ def maybe_market_to_twitter(bounty, event_name):
         return False
     if bounty.network != settings.ENABLE_NOTIFICATIONS_ON_NETWORK:
         return False
+    if settings.DEBUG and MAKE_GITCOIN_BOT_COWARDLY_IN_DEV:
```

Comment:
Do we even want to check for a global variable enabling it to make less cowardly? Or do we just check `settings.DEBUG or settings.ENV != 'prod'``

Reply:
i think we should check all three...

`MAKE_GITCOIN_BOT_COWARDLY_IN_DEV` is useful for when youre devving and want to be able to dev on notifications.  perhaps we could put it in `settings.py` and default it to `True`?

Reply:
I think putting it to setting is a better option.

Reply:
889bec1ca848c6b8f38735b33b32d339ce265dce